src/main/java/org/zalando/nakadi/repository/kafka/KafkaLocationManager.java

```diff
@@ -77,8 +80,10 @@ private static String buildBootstrapServers(final List<Broker> brokers) {
     private Properties buildKafkaProperties(final List<Broker> brokers) {
         final Properties props = new Properties();
         props.put("bootstrap.servers", buildBootstrapServers(brokers));
-        props.put("value.deserializer", "org.apache.kafka.common.serialization.StringDeserializer");
-        props.put("key.deserializer", "org.apache.kafka.common.serialization.StringDeserializer");
+        props.put(ConsumerConfig.KEY_DESERIALIZER_CLASS_CONFIG,
+                "org.apache.kafka.common.serialization.ByteArrayDeserializer");
+        props.put(ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG,
+                "org.apache.kafka.common.serialization.ByteArrayDeserializer");
```

Comment:
@adyach are these two properties used at all? As I understand they are overwritten in getKafkaConsumerProperties.
I ask this because I see that you changed it from StringDeserializer to ByteArrayDeserializer.